test/performance/odp_l2fwd.c

```diff
@@ -1976,18 +1999,6 @@ int main(int argc, char *argv[])
 
 	num_groups = gbl_args->appl.num_groups;
 
-	printf("Num worker threads: %i\n", num_workers);
-	printf("First CPU:          %i\n", odp_cpumask_first(&cpumask));
```

Comment:
This line missing in new output on purpose?

Reply:
Yes. With that change cpumask does not need to be passed to print_info(), only the cpu mask string is needed (which contains also the first cpu).